crates/edr_napi/src/instrument.rs

```diff
@@ -31,12 +33,21 @@ impl TryFrom<edr_instrument::coverage::InstrumentationResult> for Instrumentatio
 
 #[napi(object)]
 pub struct InstrumentationMetadata {
+    /// The tag that identifies the instrumented code. The tag is
```

Comment:
Some potential for confusion here that there would be one tag per (source code, source id, version) instead of many.

```suggestion
    /// The tag that identifies the instrumented code. Tags are
```